message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -111,6 +111,16 @@
         <param index="6">Empty.</param>
         <param index="7">Empty.</param>
       </entry>
+      <entry value="42006" name="MAV_CMD_FIXED_MAG_CAL_YAW">
+        <description>Magnetometer calibration based on provided known yaw. This allows for fast calibration using WMM field tables in the vehicle, given only the known yaw of the vehicle. If Latitude and longitude are both zero then use the current vehicle location.</description>
+        <param index="1" label="Yaw" units="deg">Yaw of vehicle in earth frame.</param>
```

Comment:
Can we replace this with "Yaw of vehicle relative to north" ?

Reply:
where else would it be relative too?
we really shouldn't get bogged down like this with. If someone doesn't know what yaw is when dealing with mavlink then they shouldn't be using the protocol :-)